prow/jobs/jobs.yaml

```diff
@@ -6,4 +6,26 @@ periodics:
     serviceAccountName: pre-submit-service-account
     containers:
     - image: alpine
-      command: ["/bin/date"]
\ No newline at end of file
+      command: ["/bin/date"]
+
+postsubmits:
+  #TODO(vijat@): Add support for all service controllers
+  #TODO(vijat@): Find a programmatic way to generate postsubmit jobs for all the service controllers
```

Comment:
Ideally I'd split out `jobs.yaml` into multiple yaml files that are all concatenated. I believe there is a way, as I've seen other projects do it. Possibly as part of that we could have a codegen for each service to produce its own set of job files, per.

Reply:
Looking into possible solutions for this.

Reply:
Stretch goal for now. Not necessary as part of this PR, ofc.

Reply:
Added autogeneration of jobs using jinja template.